src/compiler/utilities.ts

```diff
@@ -8253,4 +8255,60 @@ namespace ts {
         // '/// <reference no-default-lib="true"/>' directive.
         return options.skipLibCheck && sourceFile.isDeclarationFile || options.skipDefaultLibCheck && sourceFile.hasNoDefaultLib;
     }
+
+    // TODO: remove once tslint allows tsconfig.json to include lib "esnext.bigint"
+    /* @internal */
+    declare const BigInt: ((value: number | string) => number) | undefined;
+
+    // Converts a bigint literal, e.g. `0x1234n`,
+    // to its decimal representation, e.g. `4660`.
+    export function parsePseudoBigInt(stringValue: string): string {

```

Comment:
Just for prudence, I feel like this is the kind of method that'd be nice to be unittested on its own; but it's probably not strictly required since other tests should exercise it enough.

Reply:
Added a unit test: https://github.com/Microsoft/TypeScript/pull/25886/files#diff-b222e6a526e22d4babe56a51168f5fae